scripts/pdsp/config.yaml

```diff
@@ -0,0 +1,20 @@
+JobID: '61503165'
+exclude: '61503165_6'
+fields:
+  APA: !!python/tuple
+  - 0
+  - 1
+  - 2
+  - 3
+  - 4
+  - 5
+  SigProc: !!python/tuple
```

Comment:
Hi @pphuangyi. Could you please elaborate why you use explicit `tuple` type here instead of a list?

Reply:
You are right! They don't have to be tuples :)  

They are tuples because I saved the config from a python dictionary and I write them as tuples in the dictionary.